street_network/src/render.rs

```diff
@@ -65,54 +65,77 @@ impl StreetNetwork {
         Ok(output)
     }
 
-    /// Generates a more detailed GeoJSON rendering of roads and intersections.
-    pub fn to_detailed_geojson(&self, timer: &mut Timer) -> Result<String> {
```

Comment:
I got rid of this "detailed" experiment that was repeating some of what the "plain" layer does. I agree it makes sense to just have a bunch of layers representing different things, and let the caller mix them together however.